### DISCUSSION TITLE
Problem after upgrade from 1.0.20230531 with 3 Batteries

### DISCUSSION THREAD

**Discussion by pit65535** [mr-manuel/venus-os_dbus-serialbattery#106] [Q&A] [answered]
Yesterday i upgraded from ver. 1.0.20230531 (Louisvdw) to the newest stable Version via ssh. the Updatescript ended without errors:
*** dbus-serialbattery v1.4.20240928 was installed. ***
after Reboot i could not see all my 3 batteries (not in Remote Console and not using dbus-spy). After the Upgrade i only can see one Battery, 2 are missing....
All three batteries use JBD-AP21S002 BMS and are connected via USB with a USB to RS485 Interface.
Before the Upgrade everything was working fine for about one year. I have Firmware Version 3.51 installed (since 2 weeks, before was older version) on an original Cerbo GX device and use also dbus-aggregate-batteries ver. 3.3 beta.
As a workaround i rolled back to the old driver version, after that everything looks fine again.

Sur I can still use this version for a while, but I usually upgrade all my devices every year and I guess after some time it will be necessary to Update this driver..... or I must stay at this Patchlevel....

Can anybody help me what to do to get the newest stable driver working in my environment?
Any help wold be appreciated.

**Comment by mr-manuel** [comment, marked as answer]
Watching my crystal ball tells me you did not even try to check the documentation and troubleshooting section. Here you go: https://mr-manuel.github.io/venus-os_dbus-serialbattery_docs/troubleshoot/
